elements/howto-checkbox/README.md

```diff
@@ -1,5 +1,180 @@
-A `HowToCheckbox` represents a boolean option in a form. The most common type
+# `<howto-checkbox>`
+
+## Summary
+
+A `<howto-checkbox` represents a boolean option in a form. The most common type
 of checkbox is a dual-type which allows the user to toggle between two
 choices -- checked and unchecked.
 
-See: https://www.w3.org/TR/wai-aria-practices-1.1/#checkbox
+The element should attempt to self apply the attributes `role="checkbox"` and
```

Comment:
Also I’m wondering if we should use the same style guide as the comments. Meaning no “should”, but rather “The element self-applies...”